src/main/java/org/broadinstitute/hellbender/engine/filters/ReadStrandFilter.java

```diff
@@ -6,7 +6,7 @@
 import org.broadinstitute.hellbender.utils.read.GATKRead;
 
 /**
- * Keep only reads whose strand is forward or reverse
+ * Keep only reads whose strand is as specified.
```

Comment:
I think that strandness is something that it's common in the sequencing language. As an user, I understand better the meaning of a read "in the forward/reverse strand" than "aligned rightwards or leftwards". In addition, the 0x10 tag is independent of the alignment status: it just means that the sequence is reverse complemented if set, but maybe it is not aligned at all (it does not mean that is common thing, but it could happen).

I rephrased to the following due to the issue that I pointed out:

> Keep only reads whose strand is either forward (not 0x10) or reverse (0x10), as specified. The filter's default keeps forward reads (not 0x10).